plugins.d/python.d.plugin

```diff
@@ -85,7 +87,7 @@ class Job(object):
         self.recheck_every = self.job.configuration.pop('autodetection_retry')
         self.checked = False  # used in Plugin.check_job()
         self.created = False  # used in Plugin.create_job_charts()
-        if OVERRIDE_UPDATE_EVERY:
+        if self.job.update_every < int(OVERRIDE_UPDATE_EVERY):
```

Comment:
why only when it is smaller? what if it is bigger?

Reply:
Then it shouldn't be changed.

We have to override job update_every only if global bigger update_every bigger then job update interval.

Reply:
> We have to override job update_every only if global update_every bigger then job update interval.

Maybe I don't understand what is `OVERRIDE_UPDATE_EVERY`. Can you give me some more info?

Generally, when people say they want a specific `update_every` at the job level, we should respect it, independently of how it compares with the global one. Right?